patches/0001-Allow-disabling-the-partner-name.patch

```diff
@@ -1,18 +1,15 @@
-From eafcd05327894dd3f17f0ede92a78b3391b096eb Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
Are these meaningful patch edits? I've found in AWS that I can avoid adding the .patch changes unless those are intentional, which makes PRs and change history much more meaningful.

Reply:
We did have some patch conflicts in other patch files and I assumed this was a side effect of regenerating the patch set overall. 
I'm happy to discuss improvements here.